engines/gob/detection/tables_playtoons.h

```diff
@@ -113,7 +113,7 @@
 },
 {
 	{
-		"playtoons1",
+		"playtoonsdemo",
```

Comment:
The demo already has ADGF_DEMO flag, please do not specify new gameids for this.

Reply:
We have the gameid set here as an reference: https://github.com/scummvm/scummvm/commit/f93631c52a3c1a6f93c3fb9e1a12ae199acf31ce

Reply:
The gameid `playtoonsdemo` refers to a collection of the three playtoons demos (see https://scummvm.org/demos/ , so it's correct to have an individual entry.
But there's no need to rename the other `playtoon*` entries

Reply:
fixed with b616cd4de1a519ea827ce1cabe16be4660d0f614